src/mame/drivers/djboy.cpp

```diff
@@ -462,7 +423,7 @@ static INPUT_PORTS_START( djboy )
 	PORT_DIPNAME( 0x40, 0x00, DEF_STR( Demo_Sounds ) )  PORT_DIPLOCATION("SW2:7")
 	PORT_DIPSETTING(    0x40, DEF_STR( Off ) )
 	PORT_DIPSETTING(    0x00, DEF_STR( On ) )
-	PORT_DIPNAME( 0x80, 0x80, "Stereo Sound" )      PORT_DIPLOCATION("SW2:8")
+	PORT_DIPNAME( 0x80, 0x00, "Stereo Sound" )      PORT_DIPLOCATION("SW2:8")
```

Comment:
Do you have a reference for this?  Are recommended DIP switch settings given in the manual?

Reply:
On manual, not have a recommended DIP switch settings on this.
but it's doesn't effect on software, I reverted this.